era5_to_int.py

```diff
@@ -73,6 +73,83 @@ def consider(self, field, xlvl, proj, hdate, slab, intfile):
             self.t = None
             self.td = None
 
+class RHDiags:
+    """ Implements the computation of RH (relative humidity in % at pressure level)
+    from ERA5 T (pressure level temperature) and Q (specific humidity at pressure level) fields.
+    """
+    def __init__(self, levels):
+        self.spechumd = {}
+        self.t        = {}
+        self.levels   = levels
+        
+
```

Comment:
Following PEP 8 (see [Blank Lines](https://peps.python.org/pep-0008/#blank-lines)), I think we need just one blank line here.